app/src/ui/merge-branch/merge.tsx

```diff
@@ -143,7 +149,7 @@ export class Merge extends React.Component<IMergeProps, IMergeState> {
               Merge into <strong>{currentBranch ? currentBranch.name : ''}</strong>
             </Button>
           </ButtonGroup>
-          {mergeInfo}
+          {this.renderMergeInfo(invalidBranch)}
```

Comment:
It's better to continue following the existing pattern. That is, the method should return `null` when needed. For this instance `{invalidBranch()}` would follow the existing pattern.